src/domain/individualHarbor/IndividualHarborPageContainer.tsx

```diff
@@ -17,6 +17,7 @@ import HarborProperties from './harborProperties/HarborProperties';
 import LoadingSpinner from '../../common/spinner/LoadingSpinner';
 import { formatDimension } from '../../common/utils/format';
 import PierSelectHeader from './pierSelectHeader/PierSelectHeader';
+import BerthDetails from '../offer/berthDetails/BerthDetails';
```

Comment:
I tried to avoid cross-domain imports to keep each domain self-contained, my approach was: I start off with implementing the component inside its domain until it's needed in other domain, I move it out (to `common` or `domain/cards`?) I'd like to chat about it with @TatuArvela and @santtul to follow the same convention.

Reply:
Whoops, I meant to actually move this to common. But yes, the question is where's the best place to move it?

Reply:
It makes sense to have them under `common`, but I don't really know, because according to the [practices we're following](https://github.com/City-of-Helsinki/bestpractice/blob/9bb53a81bf38a8fdf3eadbddead1a319378c2cd9/docs/coding-standards.md#frontend-code-standards):
>Common: Common files are the files that aren't tied to the project's business logic. Basically, you could share these files between projects, and even maybe make them as npm packages. Though, this is not a requirement for a common file.
Which I don't believe is necessarily correct, that's why I like to discuss it with you and with Tatu and Santtu as well.

Reply:
I think a very small minority of code of individual projects could truly be used across different projects. As HDS and the new utils repository is WIP, and aim to provide cross-project reusable code, I don't think that old documentation in bestpractice is still relevant.

What makes sense to me is `common` containing code that can be reused in `domain`.

Reply:
I totally agree. 👍

Reply:
As `domain/cards` already exists with similar components, moved `BerthDetails` there.